src/pybind/mgr/dashboard/rest_client.py

```diff
@@ -511,7 +511,7 @@ def func_wrapper(self, *args, **kwargs):
                 resp_structure = api_kwargs.get('resp_structure', None)
                 args_name = inspect.getargspec(func).args
                 args_dict = dict(zip(args_name[1:], args))
-                for key, val in kwargs:
+                for key, val in kwargs.items():
```

Comment:
@votdev + @p-se : is this code actually executed?

Reply:
@sebastian-philipp This seems a hidden bug that we should backport...